tests/test-definition.yaml

```diff
@@ -20,7 +21,7 @@ dimensions:
       - 3.9.2
   - name: zookeeper-latest
     values:
-      - 3.9.2
+      - 3.9.3
```

Comment:
note: This is taken care of with #657 which is tracked by https://github.com/stackabletech/docker-images/issues/1020. We shouldn't update any unrelated products, because otherwise it is very hard to keep track of the changes.

Reply:
done https://github.com/stackabletech/hdfs-operator/pull/656/commits/3fed11693f7c56f5f7be1e9c7b03a30cf9194c55